### PR TITLE
Do not indent empty lines

### PR DESCRIPTION
**Pull request by Amine-Mike**

### CODE REVIEW

File: test/wadler_test.rb [hunk 1]
```diff
@@ -644,4 +644,22 @@ def altshow(out)
       LANG
     end
   end
+
+  describe 'minimize trailing white spaces' do
+    it 'does not indent empty lines' do
+      out = Oppen::Wadler.new(width: 10)
```
Comment (stackmystack) [inline test/wadler_test.rb]:
```suggestion
  describe 'handling empty lines' do
    it 'does not indent by default' do
      out = Oppen::Wadler.new(width: 10)
```

File: test/wadler_test.rb [hunk 2]
```diff
@@ -644,4 +644,22 @@ def altshow(out)
...
+      out.group(2) {
+        out.text 'a'
+        out.break
+        out.break
+        out.text 'b'
+      }
```
Comment (stackmystack) [inline test/wadler_test.rb:656]: Also test trivial cases, where the empty line is at the beginning xor end of the stream, then both (or at all positions).

**Review by stackmystack — CHANGES_REQUESTED** [review]

**Review by stackmystack — APPROVED** [review]
